rundeckapp/grails-spa/packages/ui/src/pages/webhooks/views/WebhooksView.vue

```diff
@@ -57,6 +57,19 @@
                     <CopyBox style="max-width: 800px" v-if="!curHook.new" :content="postUrl()"/>
                     <span class="form-control fc-span-adj font-italic" style="height: auto;" v-if="curHook.new">{{$t('message.webhookPostUrlPlaceholder')}}</span>
                   </div>
+                  <div class="card-content">
+                    <label>{{ $t('message.webhookAuthLabel') }}</label>
+                    <div class="help-block"> {{$t('message.webhookGenerateSecretCheckboxHelp')}}</div>
+                    <div class="checkbox"><input type="checkbox" v-model="curHook.useAuth" @click="confirmAuthToggle" class="form-control"><label>{{ $t('message.webhookGenerateSecurityLabel') }}</label></div>
```

Comment:
Label needs a `for` attribute, does not work to click the label

Reply:
fyi: i see the same problem with the "Enabled" checkbox, so actually a general problem with checkboxes in this UI

Reply:
```suggestion
                    <div class="checkbox"><input type="checkbox" v-model="curHook.useAuth" @click="confirmAuthToggle" class="form-control" id="wh-authtoggle"><label for="wh-authtoggle">{{ $t('message.webhookGenerateSecurityLabel') }}</label></div>
```